src/Symfony/Bundle/FrameworkBundle/Resources/config/console.xml

```diff
@@ -12,5 +12,63 @@
             <tag name="kernel.event_subscriber" />
             <tag name="monolog.logger" channel="console" />
         </service>
+
+        <service id="command.about" class="Symfony\Bundle\FrameworkBundle\Command\AboutCommand">
+            <tag name="console.command" command="about" />
+        </service>
+
+        <service id="command.assets_install" class="Symfony\Bundle\FrameworkBundle\Command\AssetsInstallCommand">
+            <argument type="service" id="filesystem" />
+            <argument>%kernel.project_dir%</argument>
+            <tag name="console.command" command="assets:install" />
+        </service>
+
+        <service id="command.cache_clear" class="Symfony\Bundle\FrameworkBundle\Command\CacheClearCommand">
+            <argument type="service" id="filesystem" />
+            <argument type="service" id="cache_clearer" />
+            <argument>%kernel.cache_dir%</argument>
+            <tag name="console.command" command="cache:clear" />
+        </service>
+
+        <service id="command.cache_pool_clear" class="Symfony\Bundle\FrameworkBundle\Command\CachePoolClearCommand">
+            <argument type="service" id="cache.global_clearer" />
+            <argument type="service" id="service_container" />
+            <argument>%kernel.cache_dir%</argument>
+            <tag name="console.command" command="cache:pool:clear" />
+        </service>
+
+        <service id="command.cache_warmup" class="Symfony\Bundle\FrameworkBundle\Command\CacheWarmupCommand">
+            <argument type="service" id="cache_warmer" />
+            <argument>%kernel.cache_dir%</argument>
+            <tag name="console.command" command="cache:warmup" />
+        </service>
+
+        <service id="command.config_debug" class="Symfony\Bundle\FrameworkBundle\Command\ConfigDebugCommand">
+            <tag name="console.command" command="debug:config" />
+        </service>
+
+        <service id="command.config_dump_reference" class="Symfony\Bundle\FrameworkBundle\Command\ConfigDumpReferenceCommand">
+            <tag name="console.command" command="config:dump-reference" />
+        </service>
+
+        <service id="command.container_debug" class="Symfony\Bundle\FrameworkBundle\Command\ContainerDebugCommand">
+            <tag name="console.command" command="debug:container" />
+        </service>
+
+        <service id="command.event_dispatcher_debug" class="Symfony\Bundle\FrameworkBundle\Command\EventDispatcherDebugCommand">
+            <argument type="service" id="event_dispatcher" />
+            <tag name="console.command" command="debug:event-dispatcher" />
+        </service>
+
+        <service id="command.router_debug" class="Symfony\Bundle\FrameworkBundle\Command\RouterDebugCommand">
+            <argument type="service" id="router" />
```

Comment:
`RouterDebugCommand` doesnt accept null ;) i remove the definition if routing is disabled in the framework extension.